src/tools/rbd/action/Status.cc

```diff
@@ -114,6 +132,14 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
       f->dump_string("state_description", migration_status.state_description);
       f->close_section(); // migration
     }
+    if (cache_state.present) {
+        f->open_object_section("image_cache_state");
```

Comment:
Nit: you need to close the sections that you open

Reply:
Done